typescript/infra/config/environments/mainnet/interchain/abacus/verification.json

```diff
@@ -0,0 +1 @@
+"{\"bsc\":[],\"avalanche\":[],\"polygon\":[],\"celo\":[],\"arbitrum\":[],\"optimism\":[{\"name\":\"router\",\"address\":\"0x1dbb057c50820b73A5450B5a0cADf5C5cc4b362E\",\"constructorArguments\":\"000000000000000000000000d83a4f747fe80ed98839e05079b1b7fe037b1638000000000000000000000000c5d6acaafbccec6d7fd7d92f4509befce641c563\",\"isProxy\":false}],\"ethereum\":[{\"name\":\"router\",\"address\":\"0x1dbb057c50820b73A5450B5a0cADf5C5cc4b362E\",\"constructorArguments\":\"000000000000000000000000ea87ae93fa0019a82a727bfd3ebd1cfca8f64f1d00000000000000000000000017e216fbb22df4ef8a6640ae9cb147c92710ac84\",\"isProxy\":false}]}"
```

Comment:
Can we make this an actual JSON object? The generation should be fixed now on `main`